Code/Tools/SceneAPI/SceneData/Behaviors/ImportGroup.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <SceneAPI/SceneCore/Components/BehaviorComponent.h>
+#include <SceneAPI/SceneCore/Events/ManifestMetaInfoBus.h>
+#include <SceneAPI/SceneCore/Events/AssetImportRequest.h>
+
+namespace AZ
+{
+    namespace SceneAPI
+    {
+        namespace Behaviors
```

Comment:
This can use a nested namespace to reduce whitespace
```suggestion
namespace AZ::SceneAPI::Behaviors
```